src/app-layout/visual-refresh-toolbar/multi-layout.ts

```diff
@@ -69,15 +69,18 @@ function mergeProps(ownProps: SharedProps, additionalProps: ReadonlyArray<Partia
   return Object.keys(toolbar).filter(key => key !== 'ariaLabels').length > 0 ? toolbar : null;
 }
 
-export function useMultiAppLayout(props: SharedProps) {
+export function useMultiAppLayout(props: SharedProps, isEnabled: boolean) {
```

Comment:
why add this as a new argument rather than to `props`? as far as I see we already have values within props (e.g. `forceDeduplicationType`) that have a similar type of function

Reply:
Because props are "props" they are coming as component configuration `<AppLayout forceDeduplicationType="primary">`. `isEnabled` is a state, so it is natural to keep it separate